deploy/deploy.lib

```diff
@@ -747,9 +763,10 @@ EOF
 
     populate_build_cache "${install_start}"
 
-    mkdir -p "${WORKSPACE:-.}/stacks/${what}"
-    rm -rf "${WORKSPACE:-.}/stacks/${what}/"*.xml
-    cp "${HOME}"/stack*.xml "${WORKSPACE:-.}/stacks/${what}"
+    log "moving 'test' results into ${WORKSPACE:-.}/stacks/${what}${part:+_}${part}"
+    mkdir -p "${WORKSPACE:-.}/stacks/${what}${part:+_}${part}"
+    rm -rf "${WORKSPACE:-.}/stacks/${what}${part:+_}${part}/"*.xml
+    cp "${HOME}"/stack*.xml "${WORKSPACE:-.}/stacks/${what}${part:+_}${part}"
 
     log "refreshing modules"
     shas="$(cut -d" " -f1 ${HOME}/specs.txt)"
```

Comment:
Module generation happens for the just installed software, i.e., what is present in the environment and was *explictly* requested. Here, we grab the explicitly installed hashes, and generate modules only for those, avoiding anything with what is considered the "latest" (had issues before)

And yes, on the team level there could be duplication, which would have to be avoided. But so far, every team installs only their final product/software...

Reply:
> But so far, every team installs only their final product/software...

Ok, Sounds good! Just want to avoid situations like someone installing `neuron%gcc` from other teams.